### ISSUE TITLE
a11y: Fix keyboard navigation, aria-labels, roles and focus display for page-level action menu

### ISSUE THREAD

**Issue by nwmac** [rancher/dashboard#12772]
a11y Issue: 3, 14

The page-level actions menu button (see below) does not show keyboard focus and does not response to enter/space to activate (14).

The menu should NOT open when it receives focus via the Tab key (3).

The menu items are missing `aria-label` and `role` attributes (3).

The menu for this does not support keyboard navigation - we should write in the new menu component - https://github.com/rancher/dashboard/issues/12771

![image](https://github.com/user-attachments/assets/0631b66b-8f17-448c-9a76-500989726181)

**Comment by IsaSih** [comment]
Works as expected in 
Rancher | v2.11-3d729db8a848fb2523bbc2f3f21aa5367dcdcc69-head
Dashboard | master df18acc
Rancher CLI | v2.11.0-rc.1
